src/commands/path/__test__/generate.path.test.ts

```diff
@@ -33,25 +32,20 @@ describe('GeneratePathImagery', () => {
     const metadata: PathMetadata = {
       targetBucketName: 'nz-imagery',
       category: 'rural-aerial-photos',
-      geographicDescription: undefined,
+      geographicDescription: 'North Island Weather Event',
```

Comment:
This is no longer just a geographic description. Should it be simply "description"?

Reply:
Here I'm trying to match the Stac Metadata field which is still `linz:geographic_description` whether the metadata should be changed now would be a question to check with @dwsilk 

(Most of the time it wont be an event just normal imagery?)

Reply:
The vast majority of the time it will be a geographic description, there are a handful of _different_ cases where it won't entirely be geographic in nature and in order to properly cover them we'd need to add a lot more complexity for very little benefit. We've split region and event names out into a standard list so that they are more useful metadata components, I think that's enough.